ipfs-cohost.sh

```diff
@@ -79,48 +81,51 @@ fi
 
 if [ "$1" = "ls" ]; then
   if [ $# -lt 2 ]; then
-    ipfs files ls /cohosting
+    ipfs files ls $MFS_DIR
     exit 0
   fi
 
   shift 1
 
   for domain in "$@"; do
     echo "snapshots for $domain"
-    ipfs files ls "/cohosting/$domain"
+    ipfs files ls "$MFS_DIR/$domain"
   done
   exit 0
 fi
 
 if [ "$1" = "sync" ]; then
-  ipfs files ls /cohosting | while read domain; do
+  ipfs files ls $MFS_DIR | while read domain; do
     echo -n "Syncing $domain..."
     update $domain
     echo " done!"
   done
   exit 0
 fi
 
-if [ "$1" = "gc" ]; then
-  if [ $# -eq 1 ]; then
-    ipfs files ls /cohosting | while read domain; do
-      echo -n "Cleaning $domain..."
-      remove $domain
-      ipfs files mkdir -p "/cohosting/$domain"
-      echo " done!"
+function prune () {
+  local historyLength=${1:-"1"}
+  ipfs files ls $MFS_DIR | while read domain; do
+    echo -n "Cleaning $domain..."
+    ipfs files ls "$MFS_DIR/$domain" | tail -r | tail -n "+$(($historyLength + 1))" | while read snap; do
```

Comment:
@olizilla  Is it possible to replace `tail -r` with something else?
It does not seem to be a part of `tail` in gnutils that are present in all  Linux distros (Macs use BSD version):
```console
tail: invalid option -- 'r'
Try 'tail --help' for more information.
```

Reply:
i used sort -n to do someting similar here https://github.com/filecoin-project/rust-fil-proofs/pull/950/files#diff-6654bd542f377955efd1cab8f1b581d0R104

Reply:
@olizilla yes, `sort -n` will work cross-platform, mind updating this PR?